app_feup/android/local.properties

```diff
@@ -1,5 +1,5 @@
-sdk.dir=/Users/nunolopes/Library/Android/sdk
-flutter.sdk=/Users/nunolopes/Documents/flutter
+sdk.dir=/home/reeckset/Android/Sdk
```

Comment:
These files are not being ignored which is a big problem

Reply:
Not really in this scope, but very true

Reply:
Indeed, I approved the pull request for this same reason. Just wanted to point it out